test/unit/tasks/data_migration/fix_backend_config_path_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class Tasks::FixBackendConfigPathTest < ActiveSupport::TestCase
+  fixtures :backend_api_configs
```

Comment:
I don't know how this works so I better ask... is this testing the case of oracle as well having it internally saved as `null` o not? because we should definitely test that too.